core/ordering/cosipbft/pbft/pbft_test.go

```diff
@@ -897,7 +895,7 @@ func makeTree(t *testing.T) (hashtree.Tree, kv.DB, func()) {
 	stage, err := tree.Stage(func(store.Snapshot) error { return nil })
 	require.NoError(t, err)
 
-	return stage, db, func() { os.RemoveAll(dir) }
+	return stage, db
```

Comment:
I don't see why we couldn't do the `db.Close()` in the clean function here, instead of changing it everywhere - it likely would've been more maintainable in case further clean-ups were needed 😅 
(like you did in pow_test.go)